outlines/1.3-migration

```diff
@@ -0,0 +1,31 @@
+1. Breaking changes (as we'd normally do this)
+
+2. Recommended changes : Modules [link to app structure guide]
+  1. Applications: Move all files to imports, initialize in one spot, link via `import`
+  2. Import *all* symbols from packages, including meteor core packages.
```

Comment:
Oh, I meant the opposite. If a package `api.exports()` a symbol, Meteor will whack it onto the global scope if you use the package in your app and thus it will be available in all files even if you start `import`-ing from within that file.